src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int base1;
+    private int base2;
+    private int height;
+
+    public IsoscelesTrapezoid(int base1, int base2, int height, String color) {
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+        this.setColor(color);
+    }
+
+    public int getBase1() {
+        return base1;
+    }
+
+    public int getBase2() {
+        return base2;
+    }
+
+    public int getHeight() {
+        return height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Drawing a isoscaledtrapezoid");
+    }
+
+    @Override
+    public double getArea() {
+        return (base1 + base2) / 2 * height;
```

Comment:
When calculating the area, it is a good practice to use parentheses to ensure the correct order of operations and improve code readability. For example: `return ((base1 + base2) / 2.0) * height;`. Also, to ensure floating-point division, `2.0` should be used instead of `2`.